rules/linux/execution_gcc_binary.toml

```
@@ -24,9 +24,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m
-[process where process.name == "gcc" and process.args == "-wrapper" and process.args : ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")]
-[process where process.parent.name == "gcc" and process.name : ("bash", "sh", "dash")]
+process where event.type == "start" and process.name in ("sh", "dash", "bash")  and process.parent.name == "gcc" and process.parent.args == "-wrapper" and process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")
```

format only suggestion

```suggestion
process where event.type == "start" and process.name in ("sh", "dash", "bash")  and 
  process.parent.name == "gcc" and process.parent.args == "-wrapper" and 
  process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")
```